SwiftWisdomTests/StandardLibrary/Collection/Collection+RandomTests.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Collection+RandomTests.swift
+//  SwiftWisdom
+//
+//  Created by Andrew Dolce on 11/21/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import XCTest
+@testable import SwiftWisdom
+
+class CollectionRandomTests: XCTestCase {
+
+    // This test is complex. Here is an explanation as to why:
```

Comment:
I see your point about maintainability. I'll take them out and leave in the original tests.